### DISCUSSION TITLE
Czech localize

### DISCUSSION THREAD

**Discussion by radys** [alfio-event/alf.io#1130] [translations]
Can you approve me to access the translation on poeditor?
English -> Czech (cz)

Thanks.

**Comment by cbellone** [comment]
Hi,
thank you for joining our translation project!

You should have access now, can you please confirm?

Should you have any questions / comments please let us know on this discussion board. Thanks!

**Reply by anthidote** [reply]
Hello, I would like to help with the Czech translation. Thanks!
